src/validation/contextualised_dataframe_validation.rs

```diff
@@ -54,6 +54,26 @@ pub(crate) fn validate_single_subject_id_column(
     }
 }
 
+pub(crate) fn validate_full_subject_id_column(
```

Comment:
`full` is not the best word. Make me think, that its just going through all values as oppose to just some.
Rather non_null, completeness or something.